src/app/get-involved/get-involved.component.html

```diff
@@ -0,0 +1,59 @@
+<app-header-static></app-header-static>
+<div class="container-top container align-center get-involved-container"> 
+  <div class="get-involved-title">
+  	Get Involved
+  </div>
+  <div class="get-involved-description">
+  	{{description}}
+  </div>
+  <div class="get-involved-section">
```

Comment:
You forgot to add row here. 
Note: If you use `col` then please wrap it in a `row`. @live-wire